templates/isa test/minimalistic_isa_test_v1.0.1.json

```diff
@@ -0,0 +1,143 @@
+{
+	"data": [
+		{
+			"metadata": {
+				"name": "ISA Minimalistic Test 1",
+				"group": "ISA Minimalistic Test",
+				"group_order": 1,
+				"temporary_name": "1_ISA_Test_Source",
+				"template_version": "1.0.0",
+				"isa_config": null,
+				"isa_measurement_type": null,
+				"isa_technology_type": null,
+				"isa_protocol_type": null,
+				"repo_schema_id": null,
+				"organism": "any",
+				"level": "study source"
+			},
+			"data": [
+				{
+					"iri": null,
+					"name": "Source Name",
+					"description": "Sources are considered as the starting biological material used in a study.",
+					"dataType": "String",
+					"title": true,
+					"required": true,
+					"isaTag": "source"
+				},
+				{
+					"iri": null,
+					"name": "Source Characteristic 1",
+					"description": "A characteristic of the source",
+					"dataType": "String",
+					"required": true,
+					"ontology": null,
+					"CVList": null,
+					"isaTag": "source_characteristic"
+				}
+			]
+		},
+		{
+			"metadata": {
+				"name": "ISA Minimalistic Test 2",
+				"group": "ISA Minimalistic Test",
+				"group_order": 1,
+				"temporary_name": "1_ISA_Test_Source",
+				"template_version": "1.0.0",
+				"isa_config": null,
+				"isa_measurement_type": null,
+				"isa_technology_type": null,
+				"isa_protocol_type": null,
+				"repo_schema_id": null,
+				"organism": "any",
+				"level": "study sample"
+			},
+			"data": [
+				{
```

Comment:
Best to add a `input` and a `protocol` here before the Sample below

Reply:
Done in commit [a04bb31](https://github.com/ELIXIR-Belgium/ISA-merger/pull/44/commits/a04bb31f4880d394e24eb2d256c321e3562cede2).